packages/contracts/package.json

```
@@ -23,8 +23,8 @@
   "scripts": {
     "forge": "forge",
     "build": "forge build",
-    "test": "ts-node ./scripts/runner/runner.ts",
-    "slither:run": "slither . --compile-force-framework foundry",
+    "test": "forge test --fork-url https://cloudflare-eth.com",
```

```suggestion
    "test": "ts-node ./scripts/runner/runner.ts",
```

I know that there's something wrong with what commit this is based on because the `runner.ts` script is definitely the correct answer here. @hashedMae perhaps you can spend some time looking into rebasing off of the correct commit?